bin/wrap

```diff
@@ -33,6 +33,7 @@ run_in_docker()
 
     if [ -e .wrap-docker-args ]; then
         EXTRA_DOCKER_ARGS="$(<.wrap-docker-args)"
+        EXTRA_DOCKER_ARGS=`eval "echo $EXTRA_DOCKER_ARGS"`
```

Comment:
Kind of a nit, but we could do nested $() to keep consistent with style:
 `EXTRA_DOCKER_ARGS=$(eval echo $(<.wrap-docker-args))`